src/main/java/io/leonis/subra/game/rule/DirectFreeKickRule.java

```diff
@@ -0,0 +1,31 @@
+package io.leonis.subra.game.rule;
+
+import java.util.*;
+import io.leonis.zosma.game.Rule;
+import io.leonis.subra.game.data.*;
+import io.leonis.subra.game.data.Referee.Command;
+
+/**
+ * The Class DirectFreeKickRule.
```

Comment:
Did we decide on the period at the end of the sentence yet?

Reply:
we did not have a discussion about this yet

Reply:
we usually don't have the period right? let's decide on removing it everywhere.

Reply:
we usually do have the period

Reply:
Alright, so you want to keep it? Fine by me. Could you open issues in corresponding repos?